mne/viz/topomap.py

```diff
@@ -1134,19 +1135,37 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = len(evoked.times) / 30
+        if order < 1:
+            order = 1
+        peaks = argrelmax(gfp, order=order, axis=0)[0]
+        if len(peaks) > 10:  # Find the largest 10 peaks
+            max_indices = np.argsort([gfp[idx] for idx in peaks])[-10:]
+            peaks = np.sort(peaks[max_indices])
+        times = evoked.times[peaks]
+        if len(times) == 0:
+            times = evoked.times[gfp.argmax()]
```

Comment:
@jaeilepp you should make a private function out of this so you can actually test something outside of matplotlib.

you could test the number of peaks using simulated data etc.

maybe you can also speed up test by avoiding too many topoplots while keeping high coverage.